### PR TITLE
add profile dump after each test

### PR DESCRIPTION
**Pull request by amarts**
* open ports required for graphite/carbon service
* create small file directory in one of the server too, so
  we can have profile info dumped there
* install pip graphitesend glustercli

Signed-off-by: Amar Tumballi <amarts@redhat.com>

### CODE REVIEW

File: tasks/prereq.yml [hunk 1]
```diff
@@ -148,3 +150,16 @@
...
+- name: Clone glustermetrics git repository
+  git:
+    clone: yes
+    depth: 1
+    update: yes
+    dest: "{{ glustermetrics_perf_git_dest | default('/usr/src/glustermetrics') }}"
+    repo: https://github.com/amarts/glustermetrics
+
+- name: setup glustermetrics conf file
+  shell: [ -n gmetrics.conf ] && cp gmetrics.conf.sample gmetrics.conf
```
Comment (amarts) [inline tasks/prereq.yml]: this seems to have issues. Better to use 'copy' module.

File: tasks/smallfile/appendoverwrite.yml [hunk 2]
```diff
@@ -38,3 +38,11 @@
      - create
      - append
      - overwrite
+
+- name: take profile info output.
+  shell: |
+      gluster volume profile {{ glusterfs_perf_volume }} info >> {{glusterfs_perf_resdir}}/smallfile/appendoverwrite-profile
+      exit 0
+  delegate_to: {{ glusterfs_perf_server }}
```
Comment (amarts) [inline tasks/smallfile/appendoverwrite.yml]: seeing warning for adding "" around {{}}

Comment (sac-urs) [reply tasks/smallfile/appendoverwrite.yml]: What is the warning?

**Comment by amarts** [comment]
fixes: #6

File: tasks/smallfile/appendoverwrite.yml (hunk 2 above)
Comment (amarts) [reply tasks/smallfile/appendoverwrite.yml]: we needed "" around {{ glusterfs_perf_server }} is what it said.. things are working now.

**Comment by amarts** [comment]
A test with latest code in this PR seems to work fine. Will merge it.

**Pull request merged**